client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/form-content/sms-notification.tsx

```diff
@@ -0,0 +1,25 @@
+import { ToggleControl } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+
+export default function SmsNotification() {
+	const translate = useTranslate();
+
+	return (
+		<>
+			<div className="notification-settings__toggle-container">
+				<div className="notification-settings__toggle">
+					<ToggleControl />
```

Comment:
Let's add an dummy onClick event to this as it throws this error

<img width="500" alt="Screenshot 2023-06-08 at 12 07 45 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/b58c15cb-761a-47ff-b022-cf4e8baee691">